test/OpenTelemetry.Exporter.Prometheus.HttpListener.Tests/EventSourceTest.cs

```diff
@@ -0,0 +1,29 @@
+// <copyright file="EventSourceTest.cs" company="OpenTelemetry Authors">
```

Comment:
Makes sense to move this file to the new common test project so it runs both `net462` and `net6.0` targets.